src/frontend/Sider/Behandling/Inngangsvilkår/typer/målgruppe.ts

```diff
@@ -55,14 +59,24 @@ export enum FaktiskMålgruppe {
     NEDSATT_ARBEIDSEVNE = 'NEDSATT_ARBEIDSEVNE',
     ENSLIG_FORSØRGER = 'ENSLIG_FORSØRGER',
     GJENLEVENDE = 'GJENLEVENDE',
+    SYKEPENGER_100_PROSENT = 'SYKEPENGER_100_PROSENT',
     INGEN_MÅLGRUPPE = 'INGEN_MÅLGRUPPE',
 }
 
+export const faktisktMålgruppeTilTekst: Record<FaktiskMålgruppe, string> = {
+    NEDSATT_ARBEIDSEVNE: 'Nedsatt arbeidsevne',
+    ENSLIG_FORSØRGER: 'Enslig forsørger',
+    GJENLEVENDE: 'Gjenlevende',
+    SYKEPENGER_100_PROSENT: '100% sykepenger gir ikke rett til tilleggsstønader',
```

Comment:
Syntes denne skurrer litt her, siden det er ikke den tekstlige versjonen av 100% sykepenger. 
Må ikke fikses nå, men burde egentlig det vises på måten "målgruppe: 100% sykepenger gir ikke rett til..."

Kunne evt hatt `faktisktMålgruppeTilTekst` med kun 100% og ha en funksjon `målgruppeGirRettTilTilleggsstønader` som man bruker for å vite om man skal legge på den siste teksten